test/transcoding/relationals_float.ll

```diff
@@ -62,9 +62,15 @@ entry:
   %add9 = add nsw i32 %add7, %call8
   %call10 = tail call spir_func i32 @_Z9isorderedff(float %f, float %f) #2
   %add11 = add nsw i32 %add9, %call10
+  %fcmp.ord = fcmp ord float %f, %f
+  %fcmp.ord.ext = zext i1 %fcmp.ord to i32
```

Comment:
Yes I forgot about that test. I've fixed it now. But something about OpenCL relational functions - they are happened to be represented in SPIRV in the same way as fcmp ord/uno, so my patch affects the way translator preserves this builtins on the way. My idea is that it is better to use native llvm instructions instead of OpenCL builtins where its possible. And this test demonstrates this exact idea. That's why I think both `cmp.ll and relations_float.ll/relations_half.ll should contain OpOrdered and OpUnordered related parts.

Reply:
Yes, it's fine for `relationals_float.ll` and `relationals_half.ll` to test translation from `isordered()` and `isunordered()` to `OpOrdered` and `OpUnordered` and back to `fcmp ord` and `fcmp uno`, but `relationals_float.ll` and `relationals_half.ll` should not test translation _from_ `fcmp ord` and `fcmp uno` to `OpOrdered` and `OpUnordered`: that falls within the purpose of `fcmp.ll` and should not be duplicated here.

Reply:
Oh, I've got it. Yes, I should separate this checks. Will fix it now.